CODEOWNERS

```diff
@@ -9,3 +9,7 @@
 
 # Exclude the src folder which is team specific
 /src/
+
+# Exclude forward flow file changes to common files to reduce noise.
+/src/source-manifest.json
```

Comment:
I think this is redundant with the previous line?

Reply:
yeah but I'd keep it in case we change that to add codeowners for each src/<repo>

Reply:
Yes, that was the intention here.